src/docs/asciidoc/cdc-configuration.adoc

```diff
@@ -595,6 +595,87 @@ The Eventuate Tram CDC supports `eventuate-tram` and `eventuate-local`
 
 |===
 
+==== Configuring a cleaner
+
+Cdc can automatically remove old messages from message table and received_message table.
+Example configuration:
+
+----
+EVENTUATE_CDC_CLEANER_C1_PIPELINE: P4
+EVENTUATE_CDC_CLEANER_C1_MESSAGE_CLEANING_ENABLED: "true"
+EVENTUATE_CDC_CLEANER_C1_MESSAGES_MAX_AGE_IN_SECONDS: 1
+EVENTUATE_CDC_CLEANER_C1_INTERVAL_IN_SECONDS: 1
+...
+----
+
+If pipeline is specified, pipeline database configuration is used.
+If not, please specify parameters explicitly:
+
+----
+EVENTUATE_CDC_CLEANER_C3_DATASOURCE_URL: jdbc:postgresql://postgreswalpipeline/eventuate
+EVENTUATE_CDC_CLEANER_C3_DATASOURCE_USERNAME: eventuate
+EVENTUATE_CDC_CLEANER_C3_DATASOURCE_PASSWORD: eventuate
+EVENTUATE_CDC_CLEANER_C3_DATASOURCE_DRIVER_CLASSNAME: org.postgresql.Driver
+EVENTUATE_CDC_CLEANER_C3_MESSAGE_CLEANING_ENABLED: "true"
+EVENTUATE_CDC_CLEANER_C3_MESSAGES_MAX_AGE_IN_SECONDS: 1
+EVENTUATE_CDC_CLEANER_C3_INTERVAL_IN_SECONDS: 1
+...
+----
+
+Full list of configuration properties
```

Comment:
Something seems to be missing here that explains what `EVENTUATE_CDC_CLEANER_C3` is, e.g. `eventuate.cdc.cleaner` is some root property name, `c3` is the name of a specific cleaner.

Reply:
ok, I will add